src/OpenTelemetry/Metrics/MetricPoint/MetricPoint.cs

```diff
@@ -3,13 +3,15 @@
 
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 using OpenTelemetry.Internal;
 
 namespace OpenTelemetry.Metrics;
 
 /// <summary>
 /// Represents a metric data point.
 /// </summary>
+[StructLayout(LayoutKind.Auto)]
```

Comment:
Using auto-layout together with the field type change for the two enums below reduces the struct size from 72 bytes to 64 bytes.